server/coordinator/node_picker.go

```diff
@@ -22,8 +25,9 @@ func NewRandomNodePicker() NodePicker {
 	return &RandomNodePicker{}
 }
 
-func (p *RandomNodePicker) PickNode(_ context.Context, registeredNodes []metadata.RegisteredNode) (metadata.RegisteredNode, error) {
-	now := time.Now().Unix()
+func (p *RandomNodePicker) PickNode(_ context.Context, _ storage.ShardID, registeredNodes []metadata.RegisteredNode) (metadata.RegisteredNode, error) {
```

Comment:
The `RandomNodePicker`'s pick logic is totally different from the `ConsistentNodePicker`, which may lead to a mass shard migration after the first shard assignment.

Reply:
It seems that `RandomNodePicker` is unnecessary in the case of `ConsistentHashNodePicker`, just remove it.